### PR TITLE
monitoring: add grafana-image-renderer dependencies

### PR DESCRIPTION
**Pull request by iameli**
It works!

image

### CODE REVIEW

File: monitoring/Dockerfile [hunk 1]
```diff
@@ -72,7 +72,31 @@ COPY --from=c1 /usr/share/grafana /usr/share/grafana
 COPY --from=c1 $GF_PATHS_HOME/bin/grafana-cli $GF_PATHS_HOME/bin/grafana-cli
 COPY --from=c1 $GF_PATHS_HOME/public $GF_PATHS_HOME/public
 
-RUN grafana-cli plugins install grafana-image-renderer
+# Grafana image rendering borrowed from https://github.com/grafana/grafana/blob/main/packaging/docker/custom/Dockerfile
```
Comment (victorges) [inline monitoring/Dockerfile:75]: How about a permalink?
```suggestion
# Grafana image rendering borrowed from https://github.com/grafana/grafana/blob/e0db19e74116db30d5e6b7666a2888238b4cb416/packaging/docker/custom/Dockerfile#L15
```

File: monitoring/Dockerfile [hunk 2]
```diff
@@ -72,7 +72,31 @@ COPY --from=c1 /usr/share/grafana /usr/share/grafana
...
+    ln -s /usr/lib/libnsl.so.2 /usr/lib/libnsl.so.1 && \
+    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.30-r0/glibc-2.30-r0.apk \
+      -O /tmp/glibc-2.30-r0.apk && \
+    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.30-r0/glibc-bin-2.30-r0.apk \
+      -O /tmp/glibc-bin-2.30-r0.apk && \
+    apk add --no-cache --allow-untrusted /tmp/glibc-2.30-r0.apk /tmp/glibc-bin-2.30-r0.apk && \
+    rm -f /tmp/glibc-2.30-r0.apk && \
+    rm -f /tmp/glibc-bin-2.30-r0.apk && \
+    rm -f /lib/ld-linux-x86-64.so.2 && \
+    rm -f /etc/ld.so.cache;
```
Comment (victorges) [inline monitoring/Dockerfile:86]: This doesn't seem to come from the `Dockerfile` you mentioned. 

Maybe include the src for this one here as well and/or move the "borrowed from" comment to the command below?

**Review by victorges — APPROVED** [review]
LGTM!

For the future: I wonder if it would be easier to invert the logic, starting `FROM` that Grafana image you referenced and installing the `node` tools instead 😆 
Or maybe some kind of multi-build, copying only what we need from each base (hard with all the dependencies I guess).

**Comment by victorges** [comment]
Also unrelated question: any way to include the region name in the alert title/description somehow?

File: monitoring/Dockerfile (hunk 2 above)
Comment (iameli) [reply monitoring/Dockerfile:86]: Done 631c51c

File: monitoring/Dockerfile (hunk 1 above)
Comment (iameli) [reply monitoring/Dockerfile]: Done 631c51c

**Comment by iameli** [comment]
> Also unrelated question: any way to include the region name in the alert title/description somehow?

Yeah, looks like there's a way to prepend a message, I'll look into it

**Pull request merged**